src/modules/addScore.js

```diff
@@ -0,0 +1,21 @@
+// import { addLocalStorage } from './localStorage.js';
+import Displaying from './displayScore.js';
+import { add } from '../data/api.js';
+
+const user = document.querySelector('section .add-score form input#name');
+const score = document.querySelector('section .add-score form input#score');
+const displaying = new Displaying();
+const addNewScore = () => {
+  if (user.value.trim() !== '' && score.value.trim() !== '') {
+    const obj = {
+      user: user.value,
+      score: score.value,
+    };
+    user.value = '';
+    score.value = '';
+    add(obj);
+    displaying.displayScore(obj);
+  }
+};
+
```

Comment:
![readme](https://user-images.githubusercontent.com/26597432/202421473-39839509-f31d-41b3-99f9-5e225027e91c.png)
- [ ] As previous reviewer suggested that you don't have anything in your readme file as you can see in the image above :arrow_up: Kindly make a descriptive readme using [microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) :+1:

Reply:
Hi @HAMMAS-SALEEM 
the readme file already has information
![image](https://user-images.githubusercontent.com/78756282/202441449-3a329f38-f67c-460e-ba79-db7c8828d476.png)

Reply:
all just copy it from the div branch and put it in this branch